modules/get-started/pages/licensing.adoc

```diff
@@ -1,31 +1,35 @@
 = Enterprise Licensing 
 :description: Learn how to purchase and apply an Enterprise Edition license.
 
-Anyone can use the Certified and Community connectors, but you need an Enterprise Edition license to use Enterprise connectors. 
+Anyone can use the Certified and Community connectors, but you need an Enterprise Edition license to use Enterprise connectors.
 
 == Purchase an Enterprise Edition license
 
 You can purchase and download a license from the https://www.redpanda.com/upgrade[Redpanda website^].
 
 == Apply a license key to Redpanda Connect
 
-Once you have downloaded a license key, you can apply it in the following ways:
+Once you have downloaded a license key, you can apply it in the following ways. 
```

Comment:
I think these two statements should be in reverse order. The first statement ends "In the following ways", but then there's a few sentences in between that and the actual ways. So maybe reverse order here.

Reply:
Updated.